src/scripts/main.js

```diff
@@ -21,7 +21,23 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
-}
+  
+    const ul = document.createElement('ul');
+    ul.textContent = data.keys;
```

Comment:
The use of 'data.keys' is incorrect. You should use 'Object.keys(data)' to get an array of the keys from the 'data' object.